src/include/stir/ProjDataInfoCylindricalNoArcCorr.inl

```diff
@@ -146,7 +146,7 @@ get_bin_for_det_pair(Bin& bin,
   }
   else
   {
-	bin.timing_pos_num() = -timing_pos_num;
```

Comment:
The reason we had this as such was to align it with what is positive or negative time difference.

Reply:
agreed that this has to be kept as-is.